src/reducers/shelf/filter.ts

```diff
@@ -35,7 +35,7 @@ export function filterReducer(
 
     case FILTER_TOGGLE: {
       const {filter} = action.payload;
-      const index = filterIndexOf(filters, filter.field);
+      const index = filterIndexOf(filters.concat(), filter.field);
```

Comment:
Undo

Reply:
The reason for adding the .concat() is for converting ReadonlyArray to a regular array

Reply:
It doesn't look fixed.